hveto/__main__.py

```diff
@@ -550,6 +552,28 @@ def main(args=None):
         #   only now do we actually have the new data to
         #   calculate significance drop
         if rnd.n > 1:
+            oldsigfile = os.path.join(
+                signidir,
+                '%s-HVETO_OLD_SIGNIFICANCE_TRIGS_ROUND_%d-%d-%d.txt' % (
+                    ifo, rnd.n, start, duration))
+            newsigfile = os.path.join(
+                 signidir,
+                 '%s-HVETO_NEW_SIGNIFICANCE_TRIGS_ROUND_%d-%d-%d.txt' % (
```

Comment:
Here `HVETO_{NEW,OLD}_SIGNIFANCE_TRIGS` makes more sense (to me) as `HVETO_SIGNIFICANT_CHANNELS` (these aren't really triggers.

Reply:
@duncanmmacleod Thanks for these suggested changes, the latest commit incorporates them. [Here's](https://ldas-jobs.ligo-la.caltech.edu/~siddharth.soni/hvetooutput/hvetotestfeb_20_2020/significance/) an example run based on this.